frontend/src/components/common/__snapshots__/ApiVersionInfo.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ApiVersionInfo suite Displays version component 1`] = `null`;
```

Comment:
I don't get this snapshot?

Reply:
yeah, this doesn't seem right

Reply:
once this gets cleaned up I think I'm good.